src/assertions/mod.rs

```diff
@@ -0,0 +1,163 @@
+pub mod compiled;
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq)]
+#[serde(rename_all = "snake_case")]
+pub struct Assertion {
```

Comment:
Thinking more about this, I sort of want to either suffix these with `config` or move these into a `config.rs` module.